Source/WebKit/UIProcess/WebsiteData/WebsiteDataStore.cpp

```diff
@@ -179,6 +179,7 @@ WebsiteDataStore::~WebsiteDataStore()
     allDataStores().remove(m_sessionID);
     if (m_networkProcess)
         m_networkProcess->removeSession(*this);
```

Comment:
Maybe we should be passing `std::exchange(m_completionHandlerForRemovalFromNetworkProcess, {})` as second parameter here? It would avoid having the `take*()` function that is a little awkward.